setup.cfg

```
@@ -27,7 +27,7 @@ install_requires =
     aiida-core~=2.2,<3
     Jinja2~=3.0
     aiida-quantumespresso~=4.4.0
-    aiidalab-widgets-base~=2.1
+    aiidalab-widgets-base[optimade]>=2.2.0a0
```

```suggestion
    aiidalab-widgets-base[optimade]~=2.2.0a0
```

Not sure if this change was intentional? Pinning to minor version seems safer, since we tend to do breaking changes on minor versions in AWB.